app/examples/react-17/src/components/TimeSeriesSmallErrorTest.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { TimeSeries } from '@propeldata/react-time-series'
+
+export function TimeSeriesSmallErrorTest() {
+  return (
+    <div className="p-4 border-2 bg-neutral-100 border-slate-600 rounded m-3">
+      <h2 className="text-2xl">TimeSeries Error Size</h2>
+      <div className="w-36">
+        <TimeSeries
+          styles={{
+            canvas: {
+              // width: 150,
+              // height: 150
+            }
+          }}
```

Comment:
@felipecadavid Wouldn't this break the chart?

Reply:
I'm not sure if I understand right, the idea here is breaking the chart so we are able to test the error fallback.
This part of the canvas prop is for testing with the canvas width and height instead of the container